cli/ballerina-cli/src/main/resources/cli-help/ballerina-bindgen.help

```diff
@@ -13,32 +13,34 @@ DESCRIPTION
        Generate Ballerina bridge code for Java APIs.
 
        Ballerina bindings could be generated for Java classes residing inside Java libraries
-       or for standard Java classes. The Java classes will be mapped to the Ballerina
-       classes providing a seamless Java interoperability developer experience to Ballerina users.
+       or standard Java classes. The Java classes will be mapped onto Ballerina
+       objects making the developer experience of Ballerina Java interoperability seamless.
 
-       In addition to the user-specified Java classes, partial implementations of directly
-       -dependent Java classes will also be generated by the tool. By default, the bindings for
-       each Java package will be mapped to a separate Ballerina module.
+       The directly-dependent Java classes and other required resources will be automatically
+       generated apart from the specified Java classes.
 
 
 OPTIONS
        (-cp|--classpath) <classpath>...
-           One or more comma-delimited classpaths for obtaining the jar files required for
+           One or more comma-delimited classpaths for obtaining the JAR files are required for
            generating the Ballerina bindings. This is not needed if the Ballerina bridge code
            is to be generated for standard Java classes.
 
        (-mvn|--maven) <groupId>:<artifactId>:<version>
-           Specify a Maven dependency with a colon-delimited group ID, artifact ID, and version.
+           A maven dependency with the colon-delimited groupId, artifactId, and version. 
            This would resolve the transitive dependencies and update the `Ballerina.toml` file.
 
-       (-o|--output) <output-path>
-           Generate all bindings inside a single directory. This option could be used in
-           instances where all the mappings are required inside a single module. The
-           specified directory doesn't need to be inside a Ballerina package.
+       (-o|--output) <output>
+           Location of the generated Ballerina bridge code. If this path is not specified,
+           the output will be written to the same directory from which the command is run.
+
+       (-m|--modules)
+           The flag that can be used to generate separate Ballerina module mappings for each
+           Java package inside a project, instead of generating all the bindings inside a
+           single output directory.
 
        --public
-           Set the visibility modifier of the generated binding objects to public. This flag
-           will be applicable only if the bindings are generated inside a single directory.
+           Set the visibility modifier of the generated binding objects to public.
```

Comment:
This too needs to be preserved. 

**Set the visibility modifier of the generated binding objects to public. This flag will be applicable only if the bindings are generated inside a single directory.**

Reply:
Fixed.